go_client/getting_started.adoc

```diff
@@ -12,229 +12,29 @@ toc::[]
-to match the client library version you use to the version of {product-title}
-with which you are interacting.
+Creating client-go applications provides an efficient way of programatically engaging activities to develop, build, deploy, and run pods in the same ways you might from the oc command line. Using client-go programs, you can adapt your {product-title} clusters to facilitate the CI/CD pipeline that best suits your organization.
 
-It is also recommended to use link:https://glide.sh/[Glide] to manage your Go
-libraries: if you do so, Glide will automatically ensure that {product-title}
-client library and its dependencies will be included in your project.
+For client-go applications, you use a link:https://github.com/openshift/client-go[client library] and link:https://github.com/openshift/api[API object definitions] to create applications that interact with the OpenShift REST API. The openshift/client-go library is designed to be compatible with and complementary to the kubernetes/client-go library. To get started, we recommend you follow these general steps:
 
-The following example connects to {product-title} using a pre-existing
-kubeconfig file, then lists all pods and builds in the current namespace.  Copy
-it as main.go into a new directory under your GOPATH.
+1. **Read the link:https://github.com/kubernetes/client-go/blob/master/README.md[client-go README]**: This file helps you understand the components of Kubernetes client-go, as well as which client-go versions you should use with different versions of Kubernetes. You can also see link:https://github.com/kubernetes/client-go/tree/master/examples/in-cluster-client-configuration[in-cluster] and link:https://github.com/kubernetes/client-go/tree/master/examples/out-of-cluster-client-configuration[out-of-cluster] examples of go client applications. Refer to link:https://github.com/openshift/client-go[OpenShift Client in Go] for specifics about OpenShift compatibility issues.
 
-.$GOPATH/src/gettingstarted/main.go
-[source, go]
-----
-package main
+1. **Read the link:https://github.com/openshift/client-go/blob/master/INSTALL.md[client-go INSTALL]: This helps you set up your client-go environment by vendoring (downloading) the correct code and helps you understand the tools you need to create applications.  Dependency management in particular is addressed.
 
-import (
-	"fmt"
+1. **Review link:https://github.com/kubernetes/client-go/tree/master/examples[client-go examples]**: These include examples of cluster authentication and managing resources, as well as more advanced concepts for interfacing with work queues and custom resource definitions.
 
-	buildv1client "github.com/openshift/client-go/build/clientset/versioned/typed/build/v1"
+1. **Review link:https://github.com/openshift/client-go/tree/master/examples/build[client-go build operations]**: Demonstrates basic operations for using Build resources. These include List builds, Retrieve details of a build, and Trigger a build.
```

Comment:
s/Demonstrates/This example demonstrates
s/Build resources/build resources

Add emphasis for the commands: These include \`List builds\`, \`Retrieve details of a build\`, and \`Trigger a build\`.